feat: Prevent duplicate seats PLUTO-656 (#1806)
codacy/docs: docs/account/emails.md

To manage the email addresses associated with your account and your email notifications, click on your avatar on the top right-hand corner and open **Your account**, page **Emails**.

![Email settings](images/emails-notifications.png)

## Updating your email addresses {: id="updating"}

Codacy pulls email addresses from the Git provider associated with your current session. On the **Emails** page, you can verify which email addresses are associated with your Codacy account.

To update the email addresses associated with your Codacy account, do the following:

1.  Update your email addresses on your Git provider ([GitHub](https://docs.github.com/en/enterprise-cloud@latest/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account), [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/set-email-aliases/), or [GitLab](https://docs.gitlab.com/ee/user/profile/#add-emails-to-your-user-profile)).

> 1.  Log out and log in to Codacy again.

Reviewer: I suggest a tweak to align the wording with the sentence after this one on [L18](https://github.com/codacy/docs/pull/1806/files#diff-56b412633cbfb63c07e4248554092c225fd2b007ce19fff5fc401d592c6bd4e4R18):

```suggestion
1.  Log out and log back in to Codacy.
```

!!! note
    -   If the updates are still not reflected on Codacy, navigate to the [access management](https://app.codacy.com/account/access-management) page, revoke the relevant Git provider or Google integration, then log out and back in to Codacy using the same provider.

>     -   For GitHub, Codacy automatically associates all commit email addresses from the same GitHub user ID to a single committer.

Reviewer: I'm missing some context on this one. Does this allow us to count users on GH just once, even if they never log in?

```suggestion
    -   When using GitHub, Codacy automatically associates all commit email addresses linked to the same GitHub user ID with a single committer.
```

Author: Yes! For GitHub, now we get the GitHub user id from the push webhook and merge the committers. 👍

Author: I'll apply your suggestions another way, as this comment is outdated.

> ### Setting your email address in local Git configuration {: id="git-config"}

Reviewer: I suggest shortening this and not mentioning local, since the user is not necessarily using Git on a local machine (they may be pushing from a remote server)

```suggestion
### Configuring your Git email address {: id="git-config"}
```

## Updating your email addresses {: id="updating"}

Codacy pulls email addresses from the Git provider associated with your current session. On the **Emails** page, you can verify which email addresses are associated with your Codacy account.

To update the email addresses associated with your Codacy account, do the following:

1.  Make sure you [set your Git email address](#git-config).

1.  Update your email addresses on your Git provider ([GitHub](https://docs.github.com/en/enterprise-cloud@latest/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account), [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/set-email-aliases/), or [GitLab](https://docs.gitlab.com/ee/user/profile/#add-emails-to-your-user-profile)).

1.  Log out and log back in to Codacy.

    !!! tip
        If the updates are still not reflected on Codacy, navigate to the [access management](https://app.codacy.com/account/access-management) page, revoke the relevant Git provider or Google integration, then log out and back in to Codacy using the same provider.

!!! note

>     **When using GitHub**, Codacy automatically associates all commit email addresses linked to the same GitHub user with a single committer. Thus, developers that never logged in to the Codacy app must also make sure they [set their Git email address](#git-config) and add all their email addresses to their [GitHub account](https://github.com/settings/emails).

Reviewer 2: We only do so as commits are made, we don't do it retro-actively. We should make this clear in the docs so that people don't think that emails from a 10k commit gut history will be merged.

Author: Nice catch, @lorandszakacs ! 👍 

```suggestion
    When developers commit **from GitHub**, Codacy automatically associates all email addresses linked to their GitHub user with a single Codacy committer. Thus, developers that never logged in to the Codacy app must also make sure they [set their Git email address](#git-config) and add all their email addresses to their [GitHub account](https://github.com/settings/emails).
```
